pkg/crdupdater/deployment.go

```diff
@@ -141,11 +140,11 @@ func (d *Deployment) remove() error {
 // deploymentComparator used to compare deployment attributes
 type deploymentComparator struct {
 	Image          string
-	Replicas       *int32
-	MinCPU         *resource.Quantity
-	MaxCPU         *resource.Quantity
-	MinMem         *resource.Quantity
-	MaxMem         *resource.Quantity
+	Replicas       int32
+	MinCPU         string
```

Comment:
Cool, that this worked 😄 .  Actually, if we used [`semantic.DeepEqual`](https://godoc.org/k8s.io/apimachinery/pkg/api/equality), we could just use `resource.Quantity` (there is a comparator for it), see [this specifically](https://github.com/kubernetes/apimachinery/blob/kubernetes-1.16.3/pkg/api/equality/semantic.go#L31) 🙃